hack/update-codegen.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo $GOPATH/pkg/mod/k8s.io/code-generator@v0.27.1)}
+
+OUTDIR="${HOME}/go/src"
+
+echo $OUTDIR
+
+CLIENTSET_NAME_VERSIONED=clientset \
+CLIENTSET_PKG_NAME=clientset \
+"${CODEGEN_PKG}/generate-groups.sh" "all" \
+  github.com/grafana/grafana/pkg/generated \
+  github.com/grafana/grafana/pkg/apis \
+  "snapshots:v0alpha1" \
```

Comment:
i have an old branch that avoids having to hard code these: https://github.com/grafana/grafana/commit/5284597de9f46858996d6eb99061de2f8316009c#diff-e885c6ff556277a34d689e152b36c1b6b7b00eb8dbd6bcb5116611e2678560b5